### PR TITLE
Address UX review comments.

### PR DESCRIPTION
**Pull request by polina-c**
See [go/notes-ux-review-for-memory-leaks-pane](https://goto.google.com/notes-ux-review-for-memory-leaks-pane)

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md [hunk 1]
```diff
@@ -39,13 +39,35 @@ myField = null;
...
+```
+
+
 
 ### Limitations
 
 The tool detects leaks for disposable and instrumented classes only (with note that the fixed leak can fix other objects too). 
 
-Some classes in Flutter framework are already instrumented. If you want your classes to be tracked, you need to make them disposable and instrument.
+Some classes in Flutter framework are already instrumented. If you want your classes to be tracked, you need to make them disposable and [instrument](#instrument).
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md:70]: nit: r / instrument / instrument them

File: packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md [hunk 2]
```diff
@@ -85,13 +107,39 @@ As Flutter widgets are instrumented, you just need to turn on the leak tracking.
 
 Invoke `ensureInitialized` and `startAppLeakTracking` before `runApp` like [the example app does](https://github.com/polina-c/spikes/blob/master/leaking_app/lib/main.dart#L7) and then follow the steps for demo app.
 
-### Add instrumentation to your classes
+### Add instrumentation to your classes <a id='instrument'></a>
 
-Invoke `startObjectLeakTracking` in constructor or initializer (that is invoked only once), and `registerDisposal` in the method `dispose`
+The tool needs to know which objects to track and it needs to know when disposal for these objects happened.
+
+To provide this information to the tool, invoke `startObjectLeakTracking` in constructor or initializer (that is invoked only once), and `registerDisposal` in the method `dispose`
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md:114]: "To provide this information to the tool for objects of a certain class invoke `startObjectLeakTracking` in the constructor..."

"... and invoke ..."

**Review by kenzieschmoll — COMMENTED** [review]
Some comments. CC @sfshaza2 for visibility as these docs will someday make it to the flutter website. For context, Shams, these are the start of some instructions for using a new tool in DevTools. We are in the phase of beginning testing on the prototype, so these docs will likely evolve and change as we continue to develop the tool. 

So at some point, we'll kindly ask you to work your magic on these :)

**Review by kenzieschmoll — APPROVED** [review]

**Review by sfshaza2 — COMMENTED** [review]
@polina-c, @kenzieschmoll, I didn't see this before it was merged, but I would like to make some edits. What is the best way for me to do that?

**Comment by polina-c** [comment]
> @polina-c, @kenzieschmoll, I didn't see this before it was merged, but I would like to make some edits. What is the best way for me to do that?

Shams, I will follow up on this with you.

**Pull request merged**